console.js

```diff
@@ -0,0 +1,37 @@
+const repl = require('repl'),
+  fs = require('fs'),
+  pjson = require('./package.json'),
+  orm = require('./app/orm'),
+  testFolder = './tests/';
```

Comment:
This is unused.
BTW, can we add a linter rule to make us remove unused variables?

Reply:
It's been turned off in the `.eslintrc` : https://eslint.org/docs/rules/no-unused-vars

Reply:
Yes, it's turned off. If you like I can open another PR turning it on and fixing the problem in every file.

Reply:
Yes, please! That would be great!